modules/store/src/reducer_manager.ts

```diff
@@ -93,12 +93,17 @@ export class ReducerManager extends BehaviorSubject<ActionReducer<any, any>>
   private updateReducers(featureKeys: string[]) {
     this.next(this.reducerFactory(this.reducers, this.initialState));
 
-    featureKeys.forEach(feature => {
+    if (featureKeys.length === 1) {
       this.dispatcher.next(<Action>{
         type: UPDATE,
-        feature,
+        feature: featureKeys[0],

```

Comment:
There should be an array dispatched either way, with one more feature keys in the array.

Reply:
👍 got it, so always a `features` array. I'll update it tomorrow.